app/jobs/publish_feed_job.rb

```diff
@@ -9,7 +9,25 @@ class PublishFeedJob < ApplicationJob
   attr_accessor :podcast, :episodes, :rss, :put_object, :copy_object
 
   def perform(podcast)
-    podcast.feeds.each { |feed| save_file(podcast, feed) }
+    podcast.feeds.each { |feed| publish_feed(podcast, feed) }
+  end
+
+  def publish_feed(podcast, feed)
+    publish_apple(feed)
```

Comment:
The hook into the publishing pipeline.

This method call does nothing unless the apple credentials are set up for the feed in question. And this feed must be on the "public" feed on the credentials.